core/src/main/java/com/google/googlejavaformat/java/JavaOutput.java

```diff
@@ -387,7 +393,7 @@ public String toString() {
     return MoreObjects.toStringHelper(this)
         .add("iLine", iLine)
         .add("lastK", lastK)
-        .add("spacesPending", spacesPending)
+        .add("peningSpaces", spacesPending.toString().replace("\t", "\\t"))
```

Comment:
The first parameter should not have changed, right?  That is, it should be `"spacesPending"`.

Reply:
Of course. Fixed that.